pre_commit_hooks/markdown_links_test.py

```diff
@@ -61,6 +61,41 @@ def test_punctuation(self) -> None:
         )
         self.assertListEqual([], links)
 
+    def test_formatting(self) -> None:
+        contents = (
+            "## Package as `package`\n\n"
+            "## Something _italicized_\n\n"
+            "## Something **bolded**\n\n"
+            "## Some _nested **chars `here`** now_\n\n"
+        )
+        headers, links = markdown_links.get_links(contents)
+        self.assertListEqual(
+            [
+                markdown_links.Header(
+                    "Package as `package`",
+                    "package-as-package",
+                    2
+                ),
+                markdown_links.Header(
+                    "Something _italicized_",
+                    "something-italicized",
+                    2
+                ),
+                markdown_links.Header(
+                    "Something **bolded**",
+                    "something-bolded",
+                    2
+                ),
+                markdown_links.Header(
+                    "Some _nested **chars `here`** now_",
+                    "some-nested-chars-here-now",
+                    2
+                ),
```

Comment:
```suggestion
                markdown_links.Header(
                    "Package as `package`", "package-as-package", 2
                ),
                markdown_links.Header(
                    "Something _italicized_", "something-italicized", 2
                ),
                markdown_links.Header(
                    "Something **bolded**", "something-bolded", 2
                ),
                markdown_links.Header(
                    "Some _nested **chars `here`** now_",
                    "some-nested-chars-here-now",
                    2,
                ),
```

Formatting from `black`:
https://github.com/google/pre-commit-tool-hooks/runs/7593009586?check_suite_focus=true